assign.py

```diff
@@ -258,13 +257,15 @@ def main():
         elif not taskchain:
             era = wf.info['AcquisitionEra']
         #Set era and procstring to none for merge ACDCs inside a task chain
-        if schema["RequestType"] == "Resubmission" and wf.info["PrepID"].startswith("task") and "Merge" in schema["InitialTaskPath"].split("/")[-2]:
+        if schema["RequestType"] == "Resubmission" and wf.info["PrepID"].startswith("task") and "Merge" in schema["InitialTaskPath"].split("/")[-1]:
             era = None
             procstring = None
 
         # Must use --lfn option, otherwise workflow won't be assigned
         if options.lfn:
             lfn = options.lfn
+        elif "MergedLFNBase" in wf.info:
+            lfn = wf.info['MergedLFNBase']
```

Comment:
The best would be to have it read from the origional request and assign 
it the same way, and leave changing it as an option because by default, 
it would needs to be assigned to the same MergedLFNBase as the parent, 
and right now we need to look that up in the parent request each time we 
assign it.

Jen


On 6/17/16 8:00 AM, Paola Katherine Rozo wrote:
>
> In assign.py 
> <https://github.com/CMSCompOps/WmAgentScripts/pull/154#discussion_r67505948>:
>
> >              era = None
> >              procstring = None
> >
> >          # Must use --lfn option, otherwise workflow won't be assigned
> >          if options.lfn:
> >              lfn = options.lfn
> > +        elif "MergedLFNBase" in wf.info:
> > +            lfn = wf.info['MergedLFNBase']
>
> To avoid any mistake, this value can be found on the original request, 
> am I right? This would be very useful in the ACDC assignment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/CMSCompOps/WmAgentScripts/pull/154/files/c03e89c97b239f1761713b48b68ff1bfe37c690c#r67505948>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AEsi257P-Mqv7mpWa5PZTeJVu5Uq47qvks5qMppagaJpZM4I3pDL>.
>